tests/material/norsand_test.cc

```diff
@@ -0,0 +1,602 @@
+#include <fstream>
```

Comment:
Remove these file streams and `cmath`

Reply:
`cmath` is required to compute M from friction angle, even at the test level. `stream` has been removed.